polyfill/test/intl.mjs

```diff
@@ -213,6 +213,676 @@ describe('Intl', () => {
     });
   });
 
+  describe('Non-ISO Calendars', () => {
+    const testChineseData = new Date('2001-02-01T00:00Z').toLocaleString('en-US-u-ca-chinese', {
+      day: 'numeric',
+      month: 'numeric',
+      year: 'numeric',
+      era: 'short',
+      timeZone: 'UTC'
+    });
+    const hasOutdatedChineseIcuData = !testChineseData.endsWith('2001');
+    const itOrSkip = (id) => ((id === 'chinese' || id === 'dangi') && hasOutdatedChineseIcuData ? it.skip : it);
+    const nodeVersion = process.versions.node.split('.')[0];
+
+    it('verify that Intl.DateTimeFormat.formatToParts output matches snapshot data', () => {
+      // This test isn't testing Temporal. Instead, it's verifying that the
+      // output of Intl.DateTimeFormat.formatToParts for non-ISO calendars
+      // hasn't changed. There are a number of outstanding bugs in this output
+      // that, when fixed, will break other tests. So this test is a signal that
+      // other tests are broken because the comparison data needs to be updated,
+      // not necessarily because Temporal is broken.
+      const getLocalizedDates = (isoString) => {
+        const calendars = [
+          'iso8601',
+          'buddhist',
+          'chinese',
+          'coptic',
+          'dangi',
+          'ethioaa',
+          'ethiopic',
+          'gregory',
+          'hebrew',
+          'indian',
+          'islamic',
+          'islamic-umalqura',
+          'islamic-tbla',
+          'islamic-civil',
+          'islamic-rgsa',
+          'islamicc',
+          'japanese',
+          'persian',
+          'roc'
+        ];
+        const date = new Date(isoString);
+        return calendars
+          .map((id) => `${id}: ${date.toLocaleDateString(`en-US-u-ca-${id}`, { timeZone: 'UTC' })}`)
+          .join('\n');
+      };
+      const year2000Content = getLocalizedDates('2000-01-01T00:00Z');
```

Comment:
if only 1999 could see us now

Reply:
Oh yeah, party like it's 1999! ;-)
![image](https://user-images.githubusercontent.com/277214/106996846-9162d380-6736-11eb-89e7-2376339e7bda.png)